guides/taxation/example-tax-setups.md

```diff
@@ -0,0 +1,70 @@
+# Example tax setups
+
+*The examples in this article are for illustrative purposes only. Always defer
+to a tax professional before setting up taxation in a production environment.* 
+
+Your store's tax setup may range from simple to complicated. This article
+provides simplified examples for typical stores that need to comply with
+value-added tax regulations, sales tax regulations, or both.
+
+## Sales tax
+
+Your U.S.-based company has [tax nexus][nexus-definition] in two states: New
+York and Pennsylvania. You do not ship outside of these two states. Because of
+this, you are required to charge different tax rates to anything shipped within
+those states:
+
+- You are required to charge 5% tax on any item shipped to New York.
+- You are required to charged 6% tax on any clothing item shipped to
+	Pennsylvania and 5% tax on non-clothing items.
+
+### Zones
+
+To simplify your taxation setup, you can create two zones:
+
+- **New York (NY)**: A state-based zone that contains only the state of New
+	York.
+- **Pennsylvania (PA)**: A state-based zone that contains only the state of
+	Pennsylvania.
+
+### Tax categories
+
+In order to create your store's tax rates, you need to create two tax categories
+for your products:
+
+- **Clothing**: All the clothing items that you sell should have this tax
+	category applied.
+- **All items**: Every product on your store should have this tax category
```

Comment:
That would then be `Other items` as a product can only have one tax category, and `All items` sounds like clothing is both `Clothing` and `All items`.

Reply:
Aah, you're very right. Not sure how I missed that. 😪 Thanks!